net/cloudflared/Portfile

```diff
@@ -0,0 +1,440 @@
+                        size    161276 \
+                    github.com/caddyserver/caddy \
+                        lock    v1.0.5 \
+                        rmd160  f6f2653262988c6f7e20cfef62dd8f36ff921d28 \
+                        sha256  04f18e1f74c35158bab8b9228782c49f87ab63f23272e5f2fef1d5a73ca6e661 \
+                        size    393049 \
+                    github.com/beorn7/perks \
+                        lock    v1.0.1 \
+                        rmd160  c6c5c7fd2132f01925c7fccd9d27c9d7a80f2adb \
+                        sha256  78da4421e9f9fa2ee5e3855bdd31cfb04c7e823d9c0ec385cc2c008132d98b96 \
+                        size    10874 \
+                    github.com/aws/aws-sdk-go \
+                        lock    v1.34.19 \
+                        rmd160  039ecc97dd5c841d0851f49401127179b8ca6c13 \
+                        sha256  1bec685acb474858fb1b73e98230cd856073349bbf3bf4aa9e757f8f4933791e \
+                        size    15948223 \
+                    github.com/anmitsu/go-shlex \
+                        lock    648efa622239 \
+                        rmd160  2cd39571128de9ea259f8ebafc292db77bfbc33e \
+                        sha256  ce0cf5fc33466e610f1605683f2e2bcb1e8212cece926074095a80f1326ed15f \
+                        size    3862 \
+                    github.com/alecthomas/units \
+                        lock    f65c72e2690d \
+                        rmd160  b2e546a67c8fc98bcb78645cb7432db04a959b47 \
+                        sha256  d3cf74fc50db9c23dd095994a98712431a8e29c3fc34ac958073c5d548de94a7 \
+                        size    4925 \
+                    github.com/acmacalister/skittles \
+                        lock    7423546701e1 \
+                        rmd160  a22d89b46d54236ae99985fa6d01e7d8fa3c21df \
+                        sha256  cfae06ff57fb446073d3748dcb1d8f1d334901630c83212f3b5140f846ad9727 \
+                        size    34610 \
+                    github.com/DATA-DOG/go-sqlmock \
+                        lock    v1.3.3 \
+                        rmd160  de25e427dea2af7e52daae8967112092145e279d \
+                        sha256  ad830f8ae529f39f1d1193925773368f16b1ea9b27c41ae0b2e60bad53d4597c \
+                        size    26581 \
+                    github.com/BurntSushi/toml \
+                        lock    v0.3.1 \
+                        rmd160  fb9650e2d16525153645e5547626f242f3800149 \
+                        sha256  8cc9e5dc68e247554227973d0b4e023b27bbd9ba5a26e4fb40f44743afcb35f1 \
+                        size    42087
+# Unsupported dependencies:
+# Second version of the same dep
+#                    github.com/coreos/go-oidc \
+                        lock    a93f71fdfe73 \
+                        rmd160  9c0bc380796ee019b4010681da8f888805c6381b \
+                        sha256  dabe10c5b5a5c4e06d943912d653479820f80f3386891fb56a2fa04475d068b3 \
+                        size    64026 \
```

Comment:
This should actually work because the code is referenced under two different package names, so they will be placed in different parts of the GOPATH.

The problem you probably encountered is that both deps extract to `coreos-go-oidc-$SHA1` and this causes the `${author}-${project}-*` glob to fail when moving the extracted code to the GOPATH.

There are two ways to fix this: 

1. Use git SHA1s for the `lock` value for both packages. Then the portgroup can use a `*-${SHA1}` glob to move the extracted source
2. Make sure`github.com/coreos/go-oidc` (locked to a SHA1) is listed above `gopkg.in/coreos/go-oidc.v2` (locked to a tag)